tests/libtest.sh

```diff
@@ -73,7 +73,7 @@ cleanup() {
     outfile=$4
   fi
 
-
+  rm -rf /var/lib/container-storage-setup
```

Comment:
Currently tests assume that they have the host. So if some of the files are present they don't like, they ask users to clean it up.

May be we can do the same thing and ask user to make sure /var/lib/container-storage-setup/ is empty otherwise error out?